app/Services/VocabularyServiceProvider.js

```diff
@@ -67,7 +67,7 @@ async function getGroupVocabulary(userId, groupId) {
   });
 
   if (group === 0) {
-    throw new ApiError(httpStatus.NOT_FOUND, 'no vocabulary cards found, because the group does not exist');
+    throw new ApiError(httpStatus.NOT_FOUND, 'no vocabulary cards found, because the group does not exist', 'group');
```

Comment:
`group` isn't a valid form field in this case. -> remove `group`

Reply:
done